sympy/stats/sampling/tests/test_sample_continuous_rv.py

```diff
@@ -162,19 +163,30 @@ def test_sample_continuous():
     scipy = import_module('scipy')
     if not scipy:
         skip('Scipy is not installed. Abort tests')
-    assert sample(Z) in Z.pspace.domain.set
-    sym, val = list(Z.pspace.sample().items())[0]
+
+    # XXX: Sampling with scipy under -Werror fails with
+    #
+    # RuntimeWarning: invalid value encountered in _cdf_single (vectorized)
+    #
+    # It isn't immediately clear if this is a bug in SciPy or SymPy.
+
+    with ignore_warnings(RuntimeWarning):
+        assert sample(Z) in Z.pspace.domain.set
+        sym, val = list(Z.pspace.sample().items())[0]
```

Comment:
This warrants further investigation. When I checked in the debugger it looked as if sympy was passing valid inputs to scipy so I don't know if the warning arises from a bug in the scipy code. Need to distil a reproducer that doesn't involve the layers of sympy code that make this quite convoluted to debug.